config.yaml

```diff
@@ -22,6 +20,9 @@ default:
  commit_updates: True
  push_updates: True
  force_rebuild: False
+release_notes:
+ release_notes_repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
+ release_notes_repo_branch: "master"
```

Comment:
release-notes subcommand may accept a packages_metadata_repo_url (versions.git) arg and a release_notes_repo_url (open-power-host-os.github.io.git) arg. I thought that calling one of them repo_url would be a little confusing for the command line user, but up to you guys.